### PR TITLE
Added in sitewide template

### PR DESCRIPTION
**Pull request by rvbsanjose**
Added in sitewide template

### CODE REVIEW

**Comment by kxprim** [comment]
:+1:

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,36 @@
+0 info it worked if it ends with ok
```
Comment (scottrice10) [inline npm-debug.log:1]: Delete this and add it to .gitignore

**Comment by scottrice10** [comment]
:+1:

**Comment by scottrice10** [comment]
:+1:

**Pull request merged**